src/app/get-involved/donate-blood/TabSection/data.ts

```diff
@@ -8,50 +8,281 @@ const whereToDonate: WhereToDonate[] = [
     {
         id: '1',
         title: 'Sunday',
-        description: `Dirghayu Hospital, Chabahil, Kathmandu
-                        Every Wednestitle, 11 AM to 3 PM
-                        Organizer: Lions Club Of Kathmandu City
-                        Location`,
+        description: `1. NRCS Central Blood Bank, Bhrikuti Mandap, Kathmandu
+- Every day(Sunday-Saturday), 7 AM to 7 PM
+- Organizer: Nepal Red Cross Society
+- [Location](https://goo.gl/maps/97d2GukQwyjsfJxU8)
+
```

Comment:
Can you also make the titles bolder and bigger, make the links red as in the prev website?